### ISSUE TITLE
Miss datepicker styles after Localization 

### ISSUE THREAD

**Issue by esradev** [jquery/jquery-ui#2206]
Hi there. I want to Localize datepicker inside a Wordpress plugin, called (GravityForms).
after successfully Localization the datepicker style will miss.

`/**
 * This file defines a jQuery UI datepicker localization for the Persian (Farsi) language.
 * It provides translations for various text elements and sets default options for the datepicker.
 * The file is intended to be used as a plugin for the jQuery UI datepicker.
 *
 * @see {@link https://jqueryui.com/datepicker/ |jQuery UI Datepicker}
 * @see {@link https://jqueryui.com/datepicker/#localization |Localization}
 */
(function (factory) {
    if (typeof define === 'function' && define.amd) {
        // AMD. Register as an anonymous module.
        define(['../datepicker'], factory);
    } else {
        // Browser globals
        factory(jQuery.datepicker);
    }
})(function (datepicker) {
    datepicker.regional['fa'] = {
        closeText: 'بستن',
        prevText: '&#x3C;قبلی',
        nextText: 'بعدی&#x3E;',
        currentText: 'امروز',
        monthNames: [
            'فروردين',
            'ارديبهشت',
            'خرداد',
            'تير',
            'مرداد',
            'شهريور',
            'مهر',
            'آبان',
            'آذر',
            'دي',
            'بهمن',
            'اسفند',
        ],
        monthNamesShort: [
            'فروردين',
            'ارديبهشت',
            'خرداد',
            'تير',
            'مرداد',
            'شهريور',
            'مهر',
            'آبان',
            'آذر',
            'دي',
            'بهمن',
            'اسفند',
        ],
        dayNames: [
            'يکشنبه',
            'دوشنبه',
            'سه شنبه',
            'چهارشنبه',
            'پنجشنبه',
            'جمعه',
            'شنبه',
        ],
        dayNamesShort: ['يک', 'دو', 'سه', 'چهار', 'پنج', 'جمعه', 'شنبه'],
        dayNamesMin: ['ي', 'د', 'س', 'چ', 'پ', 'ج', 'ش'],
        weekHeader: 'هفته',
        dateFormat: 'dd/mm/yy',
        firstDay: 6,
        isRTL: true,
        showMonthAfterYear: false,
        yearSuffix: '',
    };
    datepicker.setDefaults(datepicker.regional['fa']);

    return datepicker.regional['fa'];
});
`

![Screenshot 2024-01-06 153538](https://github.com/jquery/jquery-ui/assets/54488598/cc6cd306-2980-46d0-a984-3bc80c4e378a)
![Screenshot 2024-01-06 153642](https://github.com/jquery/jquery-ui/assets/54488598/765155dd-7044-4e89-b8c1-f10d500d1e6f)

**Comment by mgol** [comment]
Thanks for the report. Does the issue you describe exist when jQuery UI 1.12.1 is used or only with jQuery UI 1.13.0 or newer?

Also, please provide an isolated test case that we can run. A screenshot of a full system is not enough.

**Comment by esradev** [comment]
After posting this, I found out on further research that the problem I was having was because the Gravity Forms plugin had customized the date picker for itself, and provided [a javascript action to implement such changes](https://docs.gravityforms.com/gform_datepicker_options_pre_init/) in titles etc. have given.
Because of this, it is not a problem on the part of the date selector.

 And by updating the date picker object, my problem was solved.
thank you.
